test/e2e/providers/vsphere/vsphere.go

```diff
@@ -63,6 +65,21 @@ func newVSphereMachineProviderSpec(clusterID string) (*vsphere.VSphereMachinePro
 	}, nil
 }
 
+// getNetwork returns the network that needs to be used in the MachineSet
+func getNetwork() string {
+	// Default network for dev environment
+	networkSegment := "dev-segment"
+	if os.Getenv("OPENSHIFT_CI") == "true" {
+		// $LEASED_RESOURCE holds the network the CI cluster is in
+		networkSegment = os.Getenv("LEASED_RESOURCE")
```

Comment:
Is there documentation that can be linked to regarding this?

Reply:
The only references I found were [quota-and-leases/#directions-for-job-authors](https://docs.ci.openshift.org/docs/architecture/quota-and-leases/#directions-for-job-authors) and [step-registry#leases](https://docs.ci.openshift.org/docs/architecture/step-registry/#leases). Both did not have anything vSphere specific and linking to either of these does not make much sense. @jcpowermac might know of some other docs.